test/e2e/traces_noisy_span_filter_test.go

```diff
@@ -0,0 +1,118 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"fmt"
+	"net/http"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kyma-project/telemetry-manager/internal/otelcollector/ports"
+	kitk8s "github.com/kyma-project/telemetry-manager/test/testkit/k8s"
+	kitkyma "github.com/kyma-project/telemetry-manager/test/testkit/kyma"
+	kittrace "github.com/kyma-project/telemetry-manager/test/testkit/kyma/telemetry/trace"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/backend"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/urlprovider"
+	kittraces "github.com/kyma-project/telemetry-manager/test/testkit/otlp/traces"
+	"github.com/kyma-project/telemetry-manager/test/testkit/verifiers"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/kyma-project/telemetry-manager/test/testkit/periodic"
+)
+
+var _ = Describe("Traces Noisy Span Filter", Label("tracing"), func() {
```

Comment:
Let's make this description a proper sentence. Maybe "Filter Noisy Trace Spans"

Reply:
Updated